### PR TITLE
:sparkles: Implemented the notes card component

### PR DESCRIPTION
**Pull request by Harithmetic1**

### CODE REVIEW

File: components/reusables/TheNotesContainer.vue [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                    bounds: '#edit-quill',
+                    placeholder: "NOTE - s et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem ",
+                },
+                editing: false
+        }
+    },
+    methods:{
+        onEditorBlur(quill){
+            quill.blur();
+            if(this.content == ""){
```
Comment (felixfavour) [inline components/reusables/TheNotesContainer.vue:48]: === is safer

File: components/reusables/TheNotesContainer.vue [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      }
+    }
+}
+</script>
+
+<style lang="scss" scoped>
+    .note-container{
+        padding: 24px;
+        .note{
+            /* Tone Blue 02 */
```
Comment (felixfavour) [inline components/reusables/TheNotesContainer.vue:78]: I get you are copying code from Figma to make it faster but don't ma,e it seem like it.

**Review by felixfavour — APPROVED** [review]
Gave comments that need changes but I'd approve your PD be absent there are no breaking changes. Well done.

Please take note of the comments

File: components/reusables/TheNotesContainer.vue (hunk 1 above)
Comment (Harithmetic1) [reply components/reusables/TheNotesContainer.vue:48]: noted

File: components/reusables/TheNotesContainer.vue (hunk 2 above)
Comment (Harithmetic1) [reply components/reusables/TheNotesContainer.vue:78]: Sure, noted

**Pull request merged**